_rules/SC1-2-3-video-alternative.md

```diff
@@ -0,0 +1,183 @@
+---
+name: video with audio has audio descriptions or transcript
+rule_type: composite
+description: |
+  This rule checks video elements with audio have an alternative for the video content as audio or as text.
+
+success_criterion:
+- 1.2.3 # Audio Description or Media Alternative (Prerecorded)
+
+atomic_rules:
+- SC1-2-video-audio-description
+- SC1-2-video-transcript
+- SC1-2-video-description-track
+- SC1-2-video-media-alternative
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible), where the video contains audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [video element audio described](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-audio-description.html)
+- [video element transcript](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-transcript.html)
+- [Video element description track](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-description-track.html)
+- [Video as a media alternative for text](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-media-alternative.html)
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
```

Comment:
link poster to its definition (in the same way it's done in the video-only rules)
do this also for the atomic rules